pattern/udm/models.ts

```diff
@@ -50,6 +50,7 @@ export const {
   jsonbNullable,
 } = gm.domains;
 export const { autoIncPrimaryKey: autoIncPK } = gm.keys;
+export const { ulidPrimaryKey: ulidPrimaryKey } = gm.keys;
```

Comment:
This should not be exported via `models.ts`, it was a mistake. It should be used through `gm.keys`.